astropy/units/quantity.py

```diff
@@ -902,6 +902,52 @@ def to_value(self, unit=None, equivalencies=[]):
     to_value : Get the numerical value in a given unit.
     """)
 
+    def to_string(self,unit='human'):
+        """
+        Return value in a human readable unit.
+
+        Parameters
+        ----------
+        unit:string
+            the choice of user to get value in string or
+            to get value with appropriate SI prefix.
+        """
+        if not self.isscalar:
+            raise Exception("given quantity is not scalar")
+        from astropy import units as u
+        from astropy.units import get_current_unit_registry
+        from astropy.utils import console
+        from math import isclose
+        ureg = get_current_unit_registry()
+        units = []
+        if self.unit.is_equivalent(u.s) and self > 1 * u.s:
```

Comment:
if quantity is smaller than 1 second then it is dealt like any other unit.So ms,us,ns etc can also be used in conversion